src/Schema/Builder.php

```diff
@@ -346,7 +401,14 @@ protected function getAllCollections()
     {
         $collections = [];
         foreach ($this->connection->getDatabase()->listCollections() as $collection) {
-            $collections[] = $collection->getName();
+            $name = $collection->getName();
+
+            // Skip system collections
+            if (str_starts_with($name, 'system.')) {
```

Comment:
This will preserve system collection from being dropped 👍🏻 

Even for the gridfs bucket metadata, I think that's good to not remove this collection when `dropAllTables` is called.
https://www.mongodb.com/docs/manual/reference/system-collections/#database-specific-collections